curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a common method for hiding content in HTML?
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+visibility: hidden;
 
 ---
 
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for normal text according to WCAG?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+3:1
 
 ---
 
-Placeholder distractor 3
+5:1
 
 #### --answer--
 
-Placeholder answer
+4.5:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool can help inspect element styles in a web browser?
 
 #### --distractors--
 
-Placeholder distractor 1
+Adobe Photoshop
 
 ---
 
-Placeholder distractor 2
+Microsoft Word
 
 ---
 
-Placeholder distractor 3
+Notepad
 
 #### --answer--
 
-Placeholder answer
+Chrome DevTools
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute is used to hide content from screen readers?
 
 #### --distractors--
 
-Placeholder distractor 1
+hidden
 
 ---
 
-Placeholder distractor 2
+display: none;
 
 ---
 
-Placeholder distractor 3
+visibility: hidden;
 
 #### --answer--
 
-Placeholder answer
+aria-hidden
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for large text?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+4.5:1
 
 ---
 
-Placeholder distractor 3
+3.5:1
 
 #### --answer--
 
-Placeholder answer
+3:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which online tool checks color contrast accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+ColorZilla
 
 ---
 
-Placeholder distractor 2
+Canva
 
 ---
 
-Placeholder distractor 3
+Figma
 
 #### --answer--
 
-Placeholder answer
+Contrast Checker
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property hides an element from visual rendering?
```

Comment:
If this is referring to `display: none`, then this property removes the element from the DOM completely. There is a difference between visually hiding an element and removing it from the DOM. So while removing an element from the DOM does prevent it from being visually rendered, I think this question is giving a false impression that that is all the `display` property does.